herald/apps.py

```diff
@@ -13,4 +14,18 @@ class HeraldConfig(AppConfig):
     name = 'herald'
 
     def ready(self):
+        from .models import Notification
+        from herald import registry
+
         self.module.autodiscover()
+
+        try:
+            # add any new notifications to database.
+            for index, klass in enumerate(registry._registry):
+                Notification.objects.get_or_create(notification_class=klass.__name__)
```

Comment:
Ah, I see why this would be nice. So the user has a list of all the notification classes. But we should still have helper functions to enable/disable. And we need to make sure that the `notification_class` is the full dotted path to the class to prevent duplicates across apps. Would also be better if the user could see a prettier name for the notification rather than the class name. Maybe we should add a `name` variable to the main `BaseNotification` class.